SourceX/dx.cpp

```diff
@@ -6,7 +6,7 @@
 
 namespace dvl {
 
-#ifdef __AMIGA__
+#if defined (__AMIGA__) && !defined(__PPC__)

```

Comment:
Is there something like:
```suggestion
#if defined (__AMIGA__) && defined(__M68K__)
```
That would probably be more accurate?

Reply:
Could be, but technically no PPC system should see this: In rare cases (MOS) 68k code could be used when compiling emulation code. Don't know if that could trigger some hidden includes in MOS SDK.

Reply:
@Cowcat5150 if you compile for PPC, __M68K__ should never be set. The MorphOS GCC is 100% PPC only, so I don't know why it would enable 68k emulation.

Reply:
@MBeijer I know, I know. It's the "but if" in me. Just to clarify to others that when __AMIGA__ is defined it implies MOS too. By the same token if __MORPHOS__ is defined it implies Warpos in "our" particular SDK (you can differentiate it with WARPUP) so flags are going back and forth for what kind of system are you compiling. Not an easy planet.

Reply:
Shit the underscores don't show here...